daprdocs/content/en/developing-applications/building-blocks/workflow/howto-manage-workflow.md

```diff
@@ -56,6 +56,29 @@ To terminate your workflow with an ID `12345678`, run:
 POST http://localhost:3500/v1.0-alpha1/workflows/dapr/12345678/terminate
 ```
 
+### Pause or resume a workflow
```

Comment:
@cgillum Correct me if I'm wrong, but for pause and resume we no longer need workflow names, right? So the pause and resume calls below can omit the "OrderProcessingWorkflow" and simply use the workflow component and instance ID